components/script/dom/eventtarget.rs

```diff
@@ -61,17 +61,17 @@ use crate::script_runtime::CanGc;
 pub enum CommonEventHandler {
     EventHandler(#[ignore_malloc_size_of = "Rc"] Rc<EventHandlerNonNull>),
 
-    ErrorEventHandler(#[ignore_malloc_size_of = "Rc"] Rc<OnErrorEventHandlerNonNull>),
+    ErrorEvent(#[ignore_malloc_size_of = "Rc"] Rc<OnErrorEventHandlerNonNull>),
```

Comment:
Could you simply add the `allow` directive here as well and remove the other changes? It's a bit odd to remove `Handler` from all but one variant.

Reply:
I think it is worth discussing the specific instances in Zulip and deciding on a case-by-case basis. There could be cases where renaming makes sense instead of  `allow`.

Reply:
> I think it is worth discussing the specific instances in Zulip and deciding on a case-by-case basis. There could be cases where renaming makes sense instead of `allow`.

Okay, thank you @mukilan